_includes/layouts/docs.html

```diff
@@ -32,21 +32,3 @@ <h2>Table of Contents</h2>
         {% endif %}
 
         {{ content }}
-
-        <div class="grid-row">
-          <div class="grid-col flex-4"></div>
-          <div class="grid-col flex-1">
-            <p>
-              <a
-                href="{{ site.github_url }}/edit/{{ site.github_branch }}/{{ page.relative_path }}"
-                class="usa-link--external"
-                target="_blank"
-                >Suggest edits</a
-              >
-            </p>
-          </div>
-        </div>
```

Comment:
We want to keep everything after this `<div />`, just delete line 36-48. Same for the other below.